chromadb/config.py

```diff
@@ -54,6 +54,7 @@ class Settings(BaseSettings):
     persist_directory: str = ".chroma"
 
     chroma_server_host: Optional[str] = None
+    chroma_server_header: Optional[Dict[str,str]] = None
```

Comment:
maybe we should called this `chroma_server_header`? I think plural is more intuitive.